bin/upload

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+CONTAINER_NAME=factfinder-$(uuid)
+
+FF_SOURCE_DIR=data/seeds/factfinder/ && [[ "$1" == "-t" ]] && FF_SOURCE_DIR=tmp/
```

Comment:
this chaining of things with `&&` is foreign to me, maybe comment to make it clear what the condition is ?